SPECS/rubygem-fluentd/rubygem-fluentd.spec

```diff
@@ -53,12 +53,25 @@ gem install -V --local --force --install-dir %{buildroot}%{gemdir} --bindir %{bu
 %{_bindir}/fluent-plugin-config-format
 %{_bindir}/fluent-plugin-generate
 %{_bindir}/fluentd
+%{_bindir}/fluent-binlog-reader.lock
+%{_bindir}/fluent-ca-generate.lock
+%{_bindir}/fluent-cap-ctl.lock
+%{_bindir}/fluent-cat.lock
+%{_bindir}/fluent-ctl.lock
+%{_bindir}/fluent-debug.lock
+%{_bindir}/fluent-gem.lock
+%{_bindir}/fluent-plugin-config-format.lock
+%{_bindir}/fluent-plugin-generate.lock
+%{_bindir}/fluentd.lock
```

Comment:
I don't think we want to package the .lock file's?  Is that what other distributions have done?

Reply:
You are right. Updated by taking patches from https://src.fedoraproject.org/rpms/ruby/c/b7e197fb887200e4faaf8fae663a9df00bdc09d3?branch=rawhide